dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckerTest.java

```diff
@@ -59,47 +58,57 @@ private static Path absolutePathOfResource(String resourceName) {
 
   @Test
   public void testCoordinateToClasspath_validCoordinate() throws RepositoryException {
-    List<Path> paths = StaticLinkageChecker.coordinatesToClasspath("io.grpc:grpc-auth:1.15.1");
-    Truth.assertThat(paths).hasSize(12);
-
-    String pathsString = paths.toString();
+    List<Path> paths =
+        StaticLinkageChecker.artifactsToClasspath(
+            ImmutableList.of(new DefaultArtifact("io.grpc:grpc-auth:1.15.1")));
 
-    Truth.assertThat(pathsString).contains("io/grpc/grpc-auth/1.15.1/grpc-auth-1.15.1.jar");
-    Truth.assertThat(pathsString)
-        .contains(
-            "com/google/auth/google-auth-library-credentials/0.9.0/google-auth-library-credentials-0.9.0.jar");
+    Truth.assertThat(paths)
+        .comparingElementsUsing(PATH_FILE_NAMES)
+        .contains("grpc-auth-1.15.1.jar");
+    Truth.assertThat(paths)
+        .comparingElementsUsing(PATH_FILE_NAMES)
+        .contains("google-auth-library-credentials-0.9.0.jar");
     paths.forEach(
-        path -> {
-          Truth.assertWithMessage("Every returned path should be an absolute path")
-              .that(path.toString())
-              .startsWith("/");
-        });
+        path ->
+            Truth.assertWithMessage("Every returned path should be an absolute path")
+                .that(path.isAbsolute())
+                .isTrue());
   }
 
   @Test
   public void testCoordinateToClasspath_optionalDependency() throws RepositoryException {
     List<Path> paths =
-        StaticLinkageChecker.coordinatesToClasspath(
-            "com.google.cloud:google-cloud-bigtable:jar:0.66.0-alpha");
+        StaticLinkageChecker.artifactsToClasspath(
+            ImmutableList.of(
+                new DefaultArtifact("com.google.cloud:google-cloud-bigtable:jar:0.66.0-alpha")));
```

Comment:
I think if you pull the artifact out to a local variable, the rest will fit on on line and be easier to read.